### PR TITLE
OPT: LLRF-Linac

### PR DESCRIPTION
**Pull request by murilobalves**

### CODE REVIEW

File: tmp/script_linac.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    def calc_merit_function(self):
+        f_out = np.zeros(self._nswarm)
+
+        for i in range(0, self._nswarm):
+            for k in range(0, len(self.params)):
+                self.params[k].value = self._position[i, k]
+
+            _time.sleep(self._wait)
+            transmit = self.diag[3] / self.diag[2]
+            f_out[i] = self.p_energy * self.diag[0] +
```
Comment (fernandohds564) [inline tmp/script_linac.py]: I think you have to put a `.value` here to get the PVs values, don't you?

File: tmp/script_linac.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    def calc_merit_function(self):
+        f_out = np.zeros(self._nswarm)
+
+        for i in range(0, self._nswarm):
+            for k in range(0, len(self.params)):
+                self.params[k].value = self._position[i, k]
+
+            _time.sleep(self._wait)
+            transmit = self.diag[3] / self.diag[2]
```
Comment (fernandohds564) [inline tmp/script_linac.py]: It is clearer if you name your indices. Define at the beginning of the class the names you want:
```
ENERGY = 0
SPREAD = 1
ICT1_CHG = 2
ICT2_CHG = 3
```
and then use these definitions here.

File: tmp/script_linac.py (hunk 1 above)
Comment (murilobalves) [reply tmp/script_linac.py]: Sure! Thanks.

File: tmp/script_linac.py (hunk 2 above)
Comment (murilobalves) [reply tmp/script_linac.py]: Nice, thank you!

**Pull request merged**